storagenode/storagenodedb/infodb.go

```diff
@@ -23,7 +25,20 @@ var ErrInfo = errs.Class("infodb")
 
 // InfoDB implements information database for piecestore.
 type InfoDB struct {
-	db *sql.DB
+	db        *sql.DB
+	space     *space
+	bandwidth *bw
+}
+
+type space struct {
+	usedSpace int64
+	once      *sync.Once
+}
+
+type bw struct {
```

Comment:
use longer and clearer name, this package contains implementations for many things and these short names will start to get confusing really fast.

Reply:
updated